ddtrace/contrib/trace_utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+This module contains utility functions for writing ddtrace integrations.
```

Comment:
`trace_utils` vs `utils`?
I mean it's `ddtrace`, not `ddcookpotatoes` so we know it's it's utils lib for tracing ;)

Reply:
:shrug: we have a track record of having `util` or `utils` all over the repo, thought it wouldn't hurt to be a bit more verbose.